lightning/src/onion_message/functional_tests.rs

```diff
@@ -401,6 +429,62 @@ fn async_response_over_one_blinded_hop() {
 	pass_along_path(&nodes);
 }
 
+fn do_test_async_response_with_reply_path_over_one_blinded_hop(reply_path_succeed: bool) {
+	// Simulate an asynchronous interaction between two nodes, Alice and Bob.
+
+	let mut nodes = create_nodes(2);
+	let alice = &nodes[0];
+	let bob = &nodes[1];
+
+	// Alice receives a message from Bob with a reply path
+	let message = TestCustomMessage::Ping;
+	let path_id = Some([2; 32]);
```

Comment:
Could you move this to where it is used?

Reply:
I have moved the message, path_id, and reply_path (sent by Bob to us) together to update the test readability.
Let me know if the updated test flow looks good!
Thanks!